src/atlases.jl

```diff
@@ -216,12 +240,29 @@ function induced_basis(
     return dual_basis(M, p, A.basis)
 end
 
-"""
+@doc raw"""
     InducedBasis(vs::VectorSpaceType, A::AbstractAtlas, i)
 
 The basis induced by chart with index `i` from an [`AbstractAtlas`](@ref) `A` of vector
 space of type `vs`.
 
+For the `vs` a `TangentSpace` this works as  follows:
+
+Let ``n`` denote the dimension of the manifold ``\mathcal M``.
+
+Let the parameter ``a=φ_i(p) ∈ \mathbb R^n`` and ``j∈\{1,…,n\}``.
+We can look at the ``j``th parameter curve ``b_j(t) = a + te_j``, where ``e_j`` denotes the ``j``th unit vector.
+Using the parametrisation we obtain a curve ``c_i(t) = φ^{-1}(b_j(t))`` which fulfills ``c(0) = p``.
```

Comment:
```suggestion
Using the parametrisation we obtain a curve ``c_i(t) = φ_i^{-1}(b_j(t))`` which fulfills ``c(0) = p``.
```

Reply:
Thanks for carefully checking, at some point I noticed I used `I` twice and I must have missed this occurence.